do/loadbalancers.go

```diff
@@ -285,10 +285,11 @@ func (l *loadbalancers) waitActive(lbID string) (*godo.LoadBalancer, error) {
 	ctx, cancel := goctx.WithTimeout(goctx.TODO(), time.Second*time.Duration(l.lbActiveTimeout))
 	defer cancel()
 	ticker := time.NewTicker(time.Second * time.Duration(l.lbActiveCheckTick))
+	defer ticker.Stop()
 	for {
 		select {
 		case <-ticker.C:
-			lb, _, err := l.client.LoadBalancers.Get(context.TODO(), lbID)
+			lb, _, err := l.client.LoadBalancers.Get(goctx.TODO(), lbID)
```

Comment:
This is still wrong. We need to pass the `ctx` value from above so it's get propagated. The `TODO()` will do nothing in this case.